libs/deal-ticket/src/hooks/use-order-margin.ts

```diff
@@ -78,13 +82,18 @@ export const useOrderMargin = ({
     ).toString();
     const { makerFee, liquidityFee, infrastructureFee } =
       data.estimateOrder.fee;
+    const assetDecimals =
+      market.tradableInstrument.instrument.product.settlementAsset.decimals;
     return {
-      margin: addDecimal(margin, market.decimalPlaces),
-      totalFees: addDecimal(fees, market.decimalPlaces),
+      margin: addDecimalsNormalizeNumber(margin, assetDecimals),
```

Comment:
I looks into the flow of this value in code and it looks like it's changed back and forth from string to bigNum
It will go through `addDecimalsNormalizeNumber(margin, assetDecimals)` in here
then passed to `useOrderValidation ` in `DealTicketSteps` as `estMargin`
then passed to `useOrderMarginValidation` in `useOrderValidation` as `estMargin`
changed to big num - `const margin = toBigNum(estMargin?.margin || 0, 0)`
than changed to string - `const marginString = margin.toString();`
returned value is passed to `ValidateMargin`
and then it will go through normalizeFormatNumber

It looks like we divide by Math.pow(10, decimals) two times

Reply:
<img width="394" alt="image" src="https://user-images.githubusercontent.com/16125548/201342367-5167fd9b-9a70-4f55-a6e1-df1d0326884a.png">

Reply:
it is converted toBigNum in use-order-margin-validation `toBigNum(estMargin?.margin || 0, 0)` but with 0 decimals.

Reply:
ok. time for cleanup

Reply:
agree. that is what I am trying to do